docs/USAGE.md

```diff
@@ -3,26 +3,43 @@
 The Earthfile configuration in this directory facilitates builds of Space ROS from source code.
 The generated container image is based on Ubuntu 22.04 (Jammy)
 
-## Building the Docker Image
+## Prerequisites
 
-The [Earthly](https://earthly.dev/get-earthly) utility is required to build this image.
+The following software is required to build the Space ROS Docker image:
 
-To build the image, run:
+- [Docker](https://docs.docker.com/get-docker/)
+- [Earthly](https://earthly.dev/get-earthly)
+- [Git](https://git-scm.com/downloads)
+
+## Setup
+
+The image is built using the [Earthly](https://earthly.dev/get-earthly) utility. First, clone the Space ROS repository:
```

Comment:
This is extremely nit, but I've found docs easier to maintain/edit if we stick to a "one sentence per line" paradigm. It makes future changes much easier to read:

```suggestion
The image is built using the [Earthly](https://earthly.dev/get-earthly) utility.
First, clone the Space ROS repository:
```